crypto/fipsmodule/sha/keccak1600.c

```diff
@@ -405,10 +407,10 @@ size_t SHA3_Absorb(uint64_t A[SHA3_ROWS][SHA3_ROWS], const uint8_t *inp, size_t
 }
 
 size_t SHA3_Squeeze_hw(uint64_t A[SHA3_ROWS][SHA3_ROWS], const uint8_t *out, size_t len,
-                        size_t r);
+                        size_t r, int first);
 
-void SHA3_Squeeze(uint64_t A[SHA3_ROWS][SHA3_ROWS], uint8_t *out, size_t len, size_t r) {
-    SHA3_Squeeze_hw(A, out, len, r);
+void SHA3_Squeeze(uint64_t A[SHA3_ROWS][SHA3_ROWS], uint8_t *out, size_t len, size_t r, int first) {
+    SHA3_Squeeze_hw(A, out, len, r, first);
```

Comment:
Same comment about `first`.

Reply:
fixed in [3240910](https://github.com/aws/aws-lc/pull/1839/commits/32409105f83366884c41183fdd46d75fd57f2367)